pkg/telemetry/marketing/telemeter.go

```diff
@@ -0,0 +1,19 @@
+package marketing
+
+// Telemeter defines a common interface for telemetry gatherers.
+//go:generate mockgen-wrapper
+type Telemeter interface {
+	Start()
+	Stop()
+	Identify(props map[string]any)
+	TrackProps(event, userID string, props map[string]any)
```

Comment:
Is my understanding correct that `Identify()` is for updating "static" data based on the database state like # of records of different types, etc, while `TrackProps()` is for capturing events? If so, let's maybe rename for clarity.